src/app/plugins/cat.plugin.ts

```diff
@@ -0,0 +1,143 @@
+        
+        // command = command.replace("fact", "");
+        // if (command.includes("breeds")) {
+
+        //     message.reply(`Breeds supported: \n\`\`\`\n${this._breeds.join('\n')}\`\`\``);
+        //     return;
+
+        // } else {
+
+        //     var search = "";
+
+        //     if (command != "fact") {
+
+        //         search = command.replace("fact", "");
+
+
+        //         const index = this._breeds.findIndex((curBreed) => {
+        //             return command.includes(curBreed);
+        //         })
+
+        //         breed = `breed_id=${this._breedsID[index]}`;
+        //     }
+
+        //     await this.container.httpService
+        //     .get(`${this._API_URL}facts/${search}`)
+        //     .then((response: IHttpResponse) => {
+
+        //         message.reply('', {
+        //             files: [response.data[0].text],
+        //         });
+        //     })
+
+        //     .catch((err) => console.log(err));
+
+        // }
+
+    } else if (command == "breeds" || command == "breed") {
+        
+        //Simply return the list of suported breeds
+        message.reply(`Breeds supported: \n\`\`\`\n${this._breeds.join('\n')}\`\`\``);
+        return;
+
+    } else if (command == "breads" || command == "bread") {
+
+        // the Easter Egg, my gawd
+        message.reply('', {
+            files: [`https://www.thetoychronicle.com/wp-content/uploads/2016/05/Box-cat-Bread-cat-Rato-workroom-Kim-.jpg`],
+        });
+
+    } else if (command.includes("pic")){
+        // lets the system know that pic has been determined
+
+        command = command.replace("pic", "");
+        var breed = "";
+        var count = 1;
+
+        if (command.includes("bomb")){
+            //processes that the user wants 3 pictures
+            count = 3;
+            command = command.replace("bomb", "");
+        }
+
+        // determines that their was aditional text added on
+        if (command.trim() != "") {
+
+            //finds the breed
+            const index = this._breeds.findIndex((curBreed) => {
+                return command.includes(curBreed);
+            })
+
+            console.log(index)
+
+            // checks if the bread was found or not
+            if (index == -1) {
+                message.reply(`I could not find that breed, take this instead.  Please type, "cat breed" to get a list of compatible breeds.`);
+            } else {
+                breed = `&breed_id=${this._breedsID[index]}`;
+            }
+        }
+        
+        //recieves the according info and posts, or derps
+        await this.container.httpService
+        .get(`${this._API_URL}images/search?limit=${count}${breed}`)
+        .then((response: IHttpResponse) => {
+            for (let index = 0; index < count; index++) {
+                message.reply('', {
+                    files: [response.data[index].url],
+                });
+            }
```

Comment:
Unnecessary loop because only one pic should go out per invocation of the "cat" command.